POC-OOBE/OOBE-Sandbox/PowerToys Settings Sandbox/Services/NotificationService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
nit: not sure if all these includes are actually used in this files (like the threading one)